src/tools/spec_file_utils.ts

```diff
@@ -75,51 +61,29 @@ function getRefFilesPath(spec: Document) {
   return paths;
 }
 
-function applyAlternativeRef(spec: Document) {
-  walkSchema(spec, (obj) => {
-    if (obj.$ref) {
-      obj[ALTERNATIVE_REF_KEY] = obj.$ref;
-    }
-  });
-  return spec;
-}
-
-export function convertToLocalDefinition(spec: Document) {
-  walkSchema(spec, (obj) => {
-    if (obj.$ref) {
-      const index = obj.$ref.indexOf('#');
-      obj.$ref = obj.$ref.slice(index);
-    }
-  });
-  return spec;
-}
-
-export function getPreparedSpecFilePaths(specFiles: string[], tags: string[] = []) {
+export function getPreparedSpec(specFiles: string[], tags: string[] = []) {
   const readFiles: { [key: string]: boolean } = {};
-  const tmpDir = fs.mkdtempSync(
-    path.join(fs.realpathSync(os.tmpdir()), '__openapi_to_normalizr__'),
-  );
   const allFiles = uniq(getAllRelatedFiles(specFiles));
 
-  return specFiles.concat(allFiles).map((p) => {
-    const target = path.join(tmpDir, p);
-    mkdirp.sync(path.dirname(target));
-    const spec: Document = jsYaml.safeLoad(fs.readFileSync(p).toString());
-    if (specFiles.includes(p)) {
-      removeUnusableOperation(spec);
-    } else {
-      delete spec.paths; // 指定されたspecファイル以外のpath情報は不要
-    }
-    applyAlternativeRef(spec);
-    const schemas = spec.components && spec.components.schemas;
-    if (schemas) {
-      each(schemas, (model: Model, name) => {
-        model[MODEL_DEF_KEY] = name;
-      });
-    }
-    fs.writeFileSync(target, jsYaml.safeDump(spec));
-    return target;
-  });
+  return merge(
+    {},
+    ...specFiles.concat(allFiles).map((p) => {
+      const spec: Document = jsYaml.safeLoad(fs.readFileSync(p).toString());
+      if (specFiles.includes(p)) {
+        removeUnusableOperation(spec);
+      } else {
+        delete spec.paths; // 指定されたspecファイル以外のpath情報は不要
```

Comment:
行が長くなった場合の改行される可能性があるので、コメントが上の方がいいかなと思いました。（今回の作業スコープ外ですが）
```suggestion
        // 指定されたspecファイル以外のpath情報は不要
        delete spec.paths;
```